### ISSUE TITLE
mc rm --force: No object/version found to be removed in alias/bucket/

### ISSUE THREAD

**Issue by lucasbasquerotto** [minio/mc#3673]
## Expected behavior
I expect the command `mc rm --force` to behave like the shell command `rm -f` (`rm --force`) regarding exit codes, returning `0` (success) even if there are no files present (or alternatively, include a flag to ignore `not found` errors).

Although I can run the command like `mc rm ... ||:`, this would also ignore errors that should not be ignored, like incorrect credentials.

## Actual behavior
I receive the error `mc: <ERROR> No object/version found to be removed in alias/bucket/`

## Steps to reproduce the behavior

Like shown below, the error code of the `rm --force` command is `0` even if no file is present, but `mc rm --force` returns an error code in such case.

```shell
lucas@host:~/dev$ rm mytest; echo $?
rm: cannot remove 'mytest': No such file or directory
1
lucas@host:~/dev$ rm mytest/; echo $?
rm: cannot remove 'mytest/': No such file or directory
1
lucas@host:~/dev$ rm -rf mytest/; echo $?
0
lucas@host:~/dev$ rm -r mytest/; echo $?
rm: cannot remove 'mytest/': No such file or directory
1
lucas@host:~/dev$ rm -f mytest/; echo $?
0
lucas@host:~/dev$ rm -f mytest; echo $?
0
[...]
$ mc rm -r --force alias/bucket/path; echo $?
mc: <ERROR> No object/version found to be removed in `alias/bucket/`.
1
```

## mc --version
- mc version RELEASE.2021-03-12T03-36-59Z

## System information
- Docker container (minio/mc)

**Comment by kannappanr** [comment]
because we are throwing an error, we are returning an error code. This will require that we not check if the object or prefix exists and send the delete request to the object store. If we do it this way, there might be some applications that will break which rely of the error code. Let me discuss this internally

**Comment by lucasbasquerotto** [comment]
For me, it would be ok if an option to ignore `not found` errors was created, so as to keep backwards compatible.

**Comment by ssi444** [comment]
Please open this question again. We need at least an additional flag like "--ignorenotfound" so that mc --force returns 0 if there is no object to delete. This is the behavior expected by many.

**Comment by harshavardhana** [comment]
> Please open this question again. We need at least an additional flag like "--ignorenotfound" so that mc --force returns 0 if there is no object to delete. This is the behavior expected by many.

Will be fixed in #3868
